src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialog.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback } from 'react';
+import { FormikProps } from 'formik';
+import * as yup from 'yup';
+import moveDecimal from 'move-decimal-point';
+import { bigNumberify } from 'ethers/utils';
+import { useQuery } from '@apollo/client';
+import { ROOT_DOMAIN_ID } from '@colony/colony-js';
+
+import { pipe, mapPayload, withKey } from '~utils/actions';
+import { Address } from '~types/index';
+import { ActionTypes } from '~redux/index';
+import Dialog from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+import { SpinnerLoader } from '~core/Preloaders';
+import {
+  useColonyQuery,
+  ColonySubscribedUsersDocument,
+  useLoggedInUser,
+  useUser,
+} from '~data/index';
+
+import DialogForm from './CreatePaymentDialogForm';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+export interface FormValues {
+  fromDomain?: string;
+  toAssignee?: string;
+  amount: string;
+  tokenAddress?: Address;
+  reason?: string;
+}
+
+interface Props {
+  cancel: () => void;
+  close: (params: object) => void;
+  colonyAddress: Address;
+}
+
+const CreatePaymentDialog = ({ colonyAddress, cancel, close }: Props) => {
+  const validationSchema = yup.object().shape({
+    fromDomain: yup.number().required(),
+    toAssignee: yup.number().required(),
+    amount: yup.string().required(),
+    tokenAddress: yup.string().required(),
+    reason: yup.string().max(90),
+  });
+
+  const { walletAddress: loggedInUserWalletAddress } = useLoggedInUser();
+
+  const loggedInUser = useUser(loggedInUserWalletAddress);
+
+  const { data: colonyData } = useColonyQuery({
+    variables: { address: colonyAddress },
+  });
+
+  const { data: subscribedUsersData } = useQuery(
+    ColonySubscribedUsersDocument,
+    { variables: { colonyAddress } },
+  );
+
+  const tokens = (colonyData && colonyData.colony.tokens) || [];
+  const nativeTokenAddress = colonyData && colonyData.colony.nativeTokenAddress;
+
+  const transform = useCallback(
+    pipe(
+      mapPayload((payload) => {
+        // Find the selected token's decimals
+        const selectedToken = tokens.find(
+          (token) => token.address === payload.tokenAddress,
+        );
+        const decimals = getTokenDecimalsWithFallback(
+          selectedToken && selectedToken.decimals,
+        );
+
+        // Convert amount string with decimals to BigInt (eth to wei)
+        const amount = bigNumberify(moveDecimal(payload.amount, decimals));
+
+        return {
+          ...payload,
+          colonyAddress,
+          amount,
+          fromDomain: parseInt(payload.fromDomain, 10),
+          toAssignee: parseInt(payload.toAssignee, 10),
+        };
+      }),
+      withKey(colonyAddress),
+    ),
+    [colonyAddress, tokens],
+  );
+
+  return (
+    <ActionForm
+      initialValues={{
+        fromDomain: ROOT_DOMAIN_ID.toString(),
+        toAssignee: loggedInUser,
```

Comment:
why initial value of "toAssignee" is loggedInUser ? 🤔

Reply:
As Alicja said, the initial value for the user should be empty, not the currently logged in user.

Please set this to `undefined` or an empty string, ... or something :)